libs/SalesforceRestAPI/SalesforceRestAPITests/SalesforceRestAPITests.m

```diff
@@ -45,7 +45,11 @@ @implementation SalesforceRestAPITests
 + (void)setUp
 {
     @try {
-        [SFLogger setLogLevel:SFLogLevelDebug];
+        #if defined(DEBUG)
```

Comment:
Release binaries are never built for the test apps or test targets AFAIK. I don't think we need these conditionals in the test classes.